### PR TITLE
add angular-block-ui@0.2.2

### PR DESCRIPTION
**Pull request by Piicksarn**
**git repo url:** https://github.com/McNull/angular-block-ui
**Watch 27
  Star 231
 Fork 71**
@Amomo ,  could you help for me check out the PR, which is referenced to  #6546 ? thank you~
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Wrong filemap.

**Comment by Piicksarn** [comment]
@Amomo, i have the pr corrected! Would you mind to have a look at it?
Thank you!

**Comment by PeterDaveHello** [comment]
why to cc **Null** ?

**Comment by Piicksarn** [comment]
@PeterDaveHello, @Amomo,
sorry for my wrong tagging!
And i have it corrected now!
would you mind to have a look at the pr again? thank you!

File: ajax/libs/angular-block-ui/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "angular",
+    "gulp"
+  ],
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/McNull/angular-block-ui.git",
+    "basePath": "dist",
+    "files": [
+      "**/**.+(js|map|css)"
```
Comment (PeterDaveHello) [inline ajax/libs/angular-block-ui/package.json]: Wrong usage.

**Comment by Piicksarn** [comment]
@PeterDaveHello, @Amomo
i have it corrected!
would you mind to have a look at the pr? thank you~

**Pull request merged**